app/views/items/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>All Items</h1>
+<table>
+  <thead>
+    <tr>
+      <th>Item</th>
+      <th>Price</th>
+      <th>Supermarket</th>
+      <th>Customers Purchased</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @items.each do |item| %>
+      <tr>
+        <td><%= item.name %></td>
+        <td><%= item.price %></td>
+        <td><%= item.supermarket.name %></td>
+        <td><%= item.customers.count %></td>
```

Comment:
Remember we don't want to do mathy logic here. So I would create a method in your `item` model that is responsible for getting a customer count, then call it here on line 17.